src/test/java/uk/gov/companieshouse/extensions/api/attachments/file/FileTransferApiClientUnitTest.java

```diff
@@ -176,10 +170,9 @@ public void testDownload_GenericException() {
         when(restTemplate.execute(eq(DOWNLOAD_URI), eq(HttpMethod.GET), any(RequestCallback.class), ArgumentMatchers.<ResponseExtractor<ClientHttpResponse>>any(), any(FileTransferApiClientResponse.class)))
             .thenThrow(exception);
 
-        expectedException.expect(RestClientException.class);
-        expectedException.expectMessage(exception.getMessage());
-
-        fileTransferApiClient.download(FILE_ID, servletResponse);
+        RestClientException thrown = assertThrows(RestClientException.class, () ->
+            fileTransferApiClient.download(FILE_ID, servletResponse));
+        assertTrue(thrown.getMessage().contains(thrown.getMessage()));
```

Comment:
assertTrue(thrown.getMessage().contains(thrown.getMessage()));
should be
assertEquals(EXCEPTION_MESSAGE, thrown.getMessage());

Reply:
actually contains might be better.
assertTrue(thrown.getMessage().contains(EXCEPTION_MESSAGE));

Reply:
As above